server/src/test/java/org/opensearch/extensions/ExtensionsOrchestratorTests.java

```diff
@@ -420,6 +420,14 @@ public void testHandleExtensionRequest() throws Exception {
         ExtensionRequest localNodeRequest = new ExtensionRequest(ExtensionsOrchestrator.RequestType.REQUEST_EXTENSION_LOCAL_NODE);
         assertEquals(LocalNodeResponse.class, extensionsOrchestrator.handleExtensionRequest(localNodeRequest).getClass());
 
+        ExtensionRequest listenerFailureRequest = new ExtensionRequest(
+            ExtensionsOrchestrator.RequestType.REQUEST_EXTENSION_ACTION_LISTENER_ON_FAILURE,
+            "Test failure"
+        );
+        assertEquals(extensionsOrchestrator.handleExtensionRequest(listenerFailureRequest).getClass(), ExtensionBooleanResponse.class);
+        assertEquals(extensionsOrchestrator.listener.getResponseCount(), 1);
```

Comment:
Test should be `expected, returned value`

Reply:
I see you have marked the conversation as resolved but not push anything. Changing it back to `unresolve`.

Reply:
(Sometimes I click resolve when I make the changes on local before pushing... assuming that's the case here!)

Reply:
I mark it as resolved when I fix it locally